Gemfile.lock

```diff
@@ -1,11 +1,10 @@
-GIT
-  remote: https://git.earthdata.nasa.gov/scm/cmrarc/omniauth-urs.git
-  revision: 552d4faba52ff0a0727cd170cdaaead138d2ab74
-  tag: release/v1.0.0
+PATH
+  remote: /Users/cgokey/src/omniauth-urs
```

Comment:
this can't be correct

Reply:
yeah don't approve this yet then was trying to get ahead of it, approve the other one first and I'll create a release then it should update properly.